recipes/cards/ivr_node/ivr_node_default.story.vue

```diff
@@ -76,9 +76,12 @@
     </template>
     <template
       v-else
-      #menuItems
+      #menuItems="{ close }"
     >
-      <dt-list-item>
+      <dt-list-item
+        role="menuitem"
```

Comment:
nit: add `role="menuitem"` and `navigation-type="arrow-keys"` to ach list item in the story